sci-libs/blis/blis-0.5.2.ebuild

```diff
@@ -2,7 +2,8 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-inherit eutils
+PYTHON_COMPAT=( python{2_7,3_{4,5,6,7}} )
+inherit eutils python-any-r1
```

Comment:
Where is eutils used?

Reply:
This package invokes python for generating a header file, so we only need the `${PYTHON_DEPS}` variable from the eclass. The python eclass has nothing to do with the rest part of the package.

Reply:
Oops, I misread the question. `eutils` is not used here. It's sort of leftover from the template I used.